src/elements/common/__tests__/KeyBinder.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import KeyBinder from '../KeyBinder';
+
+describe('KeyBinder', () => {
+    let onScrollToChangeMock;
+    let getEvent;
+    let wrapper;
+
+    beforeEach(() => {
+        onScrollToChangeMock = jest.fn();
+        wrapper = shallow(
```

Comment:
We usually use a `getWrapper` helper and invoke it per test rather than relying on a global mutable variable.